### PR TITLE
Refactor dashdb migration

### PR DESCRIPTION
**Pull request by ssh24**
### Description
Refactor migration

#### Related issues
connect to https://github.com/strongloop/loopback-connector/issues/15

### CODE REVIEW

File: lib/migration.js [hunk 1]
```diff
@@ -1,342 +1,15 @@
 // Copyright IBM Corp. 2016. All Rights Reserved.
-// Node module: loopback-connector-dashdb
```
Comment (jannyHou) [inline lib/migration.js]: lol Please change it back to dashdb

File: lib/migration.js [hunk 2]
```diff
@@ -1,342 +1,15 @@
...
-    }, function done(err) {
-      if (err) {
-        return cb && cb(err);
-      }
-      var actual = true; // (changes.length === 0);
-      if (cb) cb(null, actual);
-    });
-  };
-};
+module.exports = function(DB2) {};
```
Comment (qpresley) [inline lib/migration.js]: Change to DASHDB

File: lib/migration.js (hunk 1 above)
Comment (ssh24) [inline lib/migration.js]: sorry. that was stupid mistake...

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Pull request merged**